core/src/components/checkbox/test/a11y/checkbox.e2e.ts

```diff
@@ -36,7 +36,7 @@ configs({ directions: ['ltr'] }).forEach(({ title, config, screenshot }) => {
                 font-size: 310%;
               }
             </style>
-            <ion-checkbox justify="start" checked>Checked</ion-checkbox>
```

Comment:
The `justify` property does not need to be on this test as all it does is force the Checkbox to be 100% width when it isn't necessary.